recipe-server/client/control/state/actions/actions.js

```diff
@@ -0,0 +1,43 @@
+import {
+  ACTION_RECEIVE,
+} from '../action-types';
+
+import {
+  makeApiRequest,
+} from '../requests/actions';
+
+
+function receiveAction(action) {
+  return dispatch => {
+    dispatch({
+      type: ACTION_RECEIVE,
+      action,
+    });
+  };
+}
+
+
+export function fetchAction(name) {
+  const requestId = `fetch-${name}`;
```

Comment:
This `requestId` is not part of a global namespace now, right? So should it be something  like `fetch-action-${name}`?